### PR TITLE
ci: Add actions cache to releaser actions

### PR DESCRIPTION
**Pull request by tjholm**

### CODE REVIEW

File: cloud/aws/deploy/down.go [hunk 1]
```diff
@@ -28,25 +28,6 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-type DownStreamMessageWriter struct {
```
Comment (HomelessDinosaur) [inline cloud/aws/deploy/down.go:31]: Is deleting this intentional?

File: cloud/azure/deploy/down.go [hunk 2]
```diff
@@ -25,25 +25,6 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-type DownStreamMessageWriter struct {
```
Comment (HomelessDinosaur) [inline cloud/azure/deploy/down.go:28]: Is this intentional?

Comment (tjholm) [reply cloud/azure/deploy/down.go:28]: these should be using the common message writer now

File: cloud/aws/deploy/down.go (hunk 1 above)
Comment (tjholm) [reply cloud/aws/deploy/down.go:31]: these should be using the common message writer now

**Review by HomelessDinosaur — APPROVED** [review]

File: .github/workflows/release-prod.yaml [hunk 3]
```diff
@@ -17,8 +17,7 @@ jobs:
         id: tag_version
         uses: mathieudutour/github-tag-action@v5.5
         with:
-          # Use GITHUB_TOKEN here to prevent further workflows
-          # generated on 'tag' action
+          dry_run: true
```
Comment (davemooreuws) [inline .github/workflows/release-prod.yaml:20]: Just a question, why was this set to dry_run?

**Review by davemooreuws — COMMENTED** [review]
Looks good to me, just a question on the prod release flow

File: .github/workflows/release-prod.yaml (hunk 3 above)
Comment (tjholm) [reply .github/workflows/release-prod.yaml:20]: We want the NITRIC_BOT_TOKEN to push to tag as part of the release, so the tag push action that publishes the binaries get's triggered.

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**